src/commands/scanner/rule/add.ts

```diff
@@ -59,4 +61,9 @@ export default class Add extends SfdxCommand {
     }
   }
 
+  private resolvePaths(): string[] {
+    // path.resolve() turns relative paths into absolute paths. It accepts multiple strings, but this is a trap because
+    // they'll be concatenated together. So we use .map() to call it on each path separately.
+    return this.flags.path.map(p => path.resolve(untildify(p)));
```

Comment:
Do we know if `untildify` is Windows aware? I think the concept of `~` as home directory doesn't exist in Windows. As long as this call doesn't mess up a Windows file path, we should be fine I suppose.

Reply:
I did some playing with the library in Windows, and I'm satisfied that it's safe.